### ISSUE TITLE
CF Perks Fulfillment 

### ISSUE THREAD

**Issue by halclark** [creativedisturbance/podcasts#179]
_From @heiwanalady on March 30, 2015 17:15_

T-shirt and sticker designs are completed.
Currently waiting on people to respond to the survey with their t-shirt size and shipping address. 

people who haven't responded to survey:
people who have:

_Copied from original issue: creativedisturbance/public-website#81_

**Comment by halclark** [comment]
_From @heiwanalady on April 6, 2015 20:17_

Just got a full cost estimate from the vendor today! Waiting for Roger's approval.

**Comment by halclark** [comment]
_From @porteriffic on May 8, 2015 16:11_

Already have stickers and t-shirts. Will work on putting together packages starting end of next week. Stephanie needs to talk to Nancy/Charlotte about mailing packages internationally. Will have cost estimate for final packaging in a week or two.

**Comment by halclark** [comment]
_From @heiwanalady on July 16, 2015 19:54_

This task has been 90% completed. T-Shirts and Stickers have been sent to backers who responded to survey in mid June. 

New Task is for @linamoon to make sure any "straggler" backers (those who answer the survey late) receives their perks. 
I have already discussed with her.

**Comment by linamoon** [comment]
Closing this for now; it seems that everyone that will respond to the survey has responded. Will fulfill perks if any more "stragglers" happen to respond. @coreysmart92 you happen to be a straggler - do you still want your shirt??
- Document for reference is CF Backers Email List: https://docs.google.com/spreadsheets/d/15tWk-698VIg96gUzmgRd_dtD6KRQwEG-7RZn36pQ2BU/edit#gid=0
